sdk/python/tests/example_repos/example_feature_repo_1.py

```diff
@@ -16,7 +16,9 @@
 )
 
 customer_profile_source = BigQuerySource(
-    table_ref="feast-oss.public.customers", event_timestamp_column="event_timestamp",
+    name="customer_profile_source",
```

Comment:
maybe add names to all other example feature repos as well?

Reply:
i wanted to have a mix of things with and without names too. But generally seems like most were using this one anyways?